app/timetables/models.py

```diff
@@ -57,3 +57,31 @@ class Course(ForceCapitalizeMixin,  models.Model):
 
     def __str__(self):
         return self.name
+
+
+
+class Timetable(TimestampMixin):
+    """
+    A timetable is the central entity or model of the platform.
```

Comment:
Please add a new line before line 66 after line 65 as this is a multi-line docstring. You can check out the PEP 8 standard here https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings